.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -6,18 +6,13 @@ body:
   - type: markdown
     attributes:
       value: |
-        ## Hello Axolotl!
-        **Thank you for contributing to the lakeFS project.**
-        We appreciate the time you invest in filing a bug report and created this template to help make this process easier.
-        It's really important to have all the information and context, to ensure we can properly address your issue. 
-        Please use the following references to fill out the bug report.
+        Thank you for contributing to the lakeFS project.
+        For us to properly address your issue, please fill in the following fields.
```

Maybe even fewer words?
```suggestion
        The following fields help us properly address your issue:
```